lib/certUtils.js

```diff
@@ -480,7 +487,7 @@ const certMap = new Map();
 class CertManager {
 	static addCert(certBuf) {
 		const cert = new Certificate(certBuf);
-		const serial = cert.getSerial();
+		const serial = cert.getCommonName();
```

Comment:
So my question now is: do you think that it's confusing to use the common name instead of the serial number to identify the cert in `CertManager`? I didn't want to change this in order to not break anything, however the methods of this class (for example, [getCertBySerial](https://github.com/webauthn-open-source/fido2-lib/blob/f80fef8dfa73f06f61175a96fb2abbaa93e378c3/lib/certUtils.js#L500)) seem to indicate that just the serial number should be required to identify the cert. Do you think this is something that should be changed?

Reply:
Alright, I've added those changes in CertManager. Please let me know if that's what you had in mind or if there's anything else!

Reply:
Great, that's exactly what I was thinking of. Could you do the same optional parameter and deprecation warnings for those methods? So that for example the "getIssuer" has also an optional version parameter, where none or the old version create a warning and returns the common name, while a new version actually returns the issuer.

Reply:
Ah yes, that might've lead to the confusion - all great now!